packages/babel-parser/test/fixtures/flow/type-parameter-declaration/default-invalid-1/input.js

```diff
@@ -1,3 +1 @@
-(class A {
-  foo<T = string>() {}
-});
+declare function foo<T = string>() {}
```

Comment:
Since we're in here shuffling things, can we give these tests better names so we don't have to go through this renumbering?

Reply:
i wonder how i should name it, i copied the test case from [flow fixtures](https://github.com/facebook/flow/tree/master/src/parser/test/flow/types/parameter_defaults), should i renamed it based on their test id?

Reply:
Maybe:

`default-invalid-declare-func`
`default-invalid-var-type`
`default-invalid-class-extends`

🤷‍♂